Tools/LambdaTestTool-v2/tests/Amazon.Lambda.TestTool.IntegrationTests/ApiGatewayResponseExtensionsTests.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.Lambda.APIGatewayEvents;
+using Amazon.Lambda.TestTool.IntegrationTests.Helpers;
+using Amazon.Lambda.TestTool.Models;
+using static ApiGatewayResponseTestCases;
+
+namespace Amazon.Lambda.TestTool.IntegrationTests
+{
+    [Collection("ApiGateway Integration Tests")]
+    public class ApiGatewayResponseExtensionsTests
+    {
+        private readonly ApiGatewayIntegrationTestFixture _fixture;
+
+        public ApiGatewayResponseExtensionsTests(ApiGatewayIntegrationTestFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Theory]
+        [MemberData(nameof(ApiGatewayResponseTestCases.V1TestCases), MemberType = typeof(ApiGatewayResponseTestCases))]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "xUnit1026:Theory methods should use all of their parameters")]
```

Comment:
i added this suppression so in vscode we can easily see what test is being executed by having test name parameter here